src/plugins/ep_lightship/server.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * lightship health check status for etherpad-lite
+ *
+ * This plugin is supposed to start a webserver that accepts HTTP requests on a
+ * specific port to check the status of the etherpad-lite server. You can
+ * configure it's behavior by defining the following environment variables:
+ *
+ * EP_LIGHTSHIP_DETECT_KUBERNETES: should lightship detect its environment?
+ * i.e. local or k8s (default: false).
+ *
+ * EP_LIGHTSHIP_PORT: the lightship server port (default: 9002).
+ */
+
+const lp = require('lightship');
+
+exports.start = function(hook_name, args, cb) {
```

Comment:
Add a comment to know what are the parameters ? for `cb` I have no idea.

Reply:
`cb` stands for callback